lib/most_frequent_integer.rb

```diff
@@ -3,5 +3,7 @@
 # exists, return nil.
 
 def find_most_frequent_integer(arr)
-  # put your code here
+  h = create_collection_of_integers(arr)
```

Comment:
Did you define your own method called `create_collection_of_integers`? If so, you'll need to include that in this file–otherwise Ruby returns an error (It complains that there's no such operation as `create_collection_of_integers`).